### ISSUE TITLE
Shouldn't the cursor pointer has been changed in `link-btn` when it's disabled

### ISSUE THREAD

**Issue by dinukadesilva** [Pearson-Higher-Ed/elements#232]
`<button type="button" class="pe-link--btn" disabled>Disabled link button 2.0</button>`

http://pearson-higher-ed.github.io/elements/elements/buttons/

**Comment by StommePoes** [comment]
Good catch, I agree!

**Comment by kaadams4** [comment]
@dinukadesilva Do you want to submit a pull request or should I just create a story for the pda team to fix?

**Comment by StommePoes** [comment]
I've already done it for sdk in latest PR.

**Comment by kaadams4** [comment]
I should have known :D Yeah we should probably fix it in both.

**Comment by dinukadesilva** [comment]
Great.... and another thing that i would like to suggest is, why shouldn't we use elements as a dependency in sdk rather duplicating the same code. 

Regarding elements sdk, are we expecting everyone to use react in their projects. What about angular projects and if there is a new ui framework comes in the future. Just a doubt.... in my point of view, elements is the best place to keep all css.

Rrgards,
@dinukadesilva

**Comment by StommePoes** [comment]
I think the idea was to let elements go off into the sunset, and the single copy would be in the sdk, and for people who only want the styles and no react-foobarbaz, there was supposed to be some way to just extract the elements/styles stuff. But to be honest I'm not clear how you do that exactly.

Dinuka, did you want to do the PR for the elements version of the disabled link button?

**Comment by dinukadesilva** [comment]
Yp... Im concern a bit about elements sdk

Yp. I can do the fix for sure

**Comment by kaadams4** [comment]
@dinukadesilva Yeah, elements is basically the old sdk and elements-sdk is the new one. We are supporting both right now because we have a couple of teams that aren't able to upgrade yet.

**Comment by kaadams4** [comment]
@dinukadesilva Regarding react vs Angular, we selected react because it can work with vanilla js, react, Angular, etc. However, some Angular teams have stated that they don't want to pull in react components. I have suggested to those teams that they submit Angular components to compliment our react components. I can imagine our library expanding to support multiple options for various tech stacks. However, our team is a react time right now so we will continue to work in react for now.
